hack/.vendor-helpers.sh

```diff
@@ -70,15 +70,17 @@ clean() {
 		github.com/docker/docker/integration-cli # external tests
 	)
 
-	local dockerPlatforms=( linux/amd64 $(_dockerfile_env DOCKER_CROSSPLATFORMS) )
+	local dockerPlatforms=( linux/amd64 windows/amd64 $(_dockerfile_env DOCKER_CROSSPLATFORMS) )
```

Comment:
What's the reason for this change?  `_dockerfile_env DOCKER_CROSSPLATFORMS` should be picking up `windows/amd64` already, so I'm really keen on fixing that if it isn't working properly (since that's going to cause other problems too).